packages/main/test/specs/ColorPalettePopover.spec.js

```diff
@@ -150,4 +150,44 @@ describe("ColorPalette interactions", () => {
 		assert.ok(await colorPalette.getProperty("showMoreColors"), "Check if more colors is on");
 	})
 
+	it("After selecting an item, opening the popover again should focus the selected item", async () => {
+		await browser.url(`test/pages/ColorPalettePopover.html`);
```

Comment:
Do we need to reload the page? If we can skip it it will be good. I see that we test on the same popover, but it will be good if we skip the reload

Reply:
We do not need the test reloads, instead I will add another test example.